packages/nexrender-action-encode/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nexrender/action-encode",
   "author": "inlife",
-  "version": "1.8.16",
+  "version": "1.8.17",
```

Comment:
could you remove that version bump?
repo uses lerna to manage version updates, so there is no need for that